generator/src/main.rs

```diff
@@ -280,8 +313,7 @@ impl Parser {
                             } else if name.ends_with("EXTENSION_NAME") {
                                 let value = attr(&attributes, "value").unwrap();
                                 assert_eq!(&ext_name[..], &value[1..value.len() - 1]);
-                            } else {
-                                let value = attr(&attributes, "value").unwrap();
+                            } else if let Some(value) = attr(&attributes, "value") {
```

Comment:
Does removing the `unwrap` that was previously here put us at risk of missing future additions of new cases that the generator doesn't support?

Reply:
Removing the unwrap should have come in a later commit but I missed to move it while rebasing.